README.md

```diff
@@ -41,9 +41,10 @@ This will generate a new executable named: `log-agent-<logstash_version>_<logsta
 In order to create the monasca-ui installer, you need to run the
 following command:
 ```
-./create_monasca_ui_installer.sh <version_number>
+./create_monasca_ui_installer.sh <version_number> <upper_constraint_file>
 ```
 
+You can find an example of an upper constraint file [here](https://github.com/openstack/requirements/blob/master/upper-constraints.txt).
```

Comment:
It would be better to use URL to plain text file. It would allow to use that URL directly as argument without the need of downloading the file. e.g. http://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt?h=stable/pike

Reply:
Thanks, done.